ziggurat-core-geoip/src/geoutils.rs

```diff
@@ -0,0 +1,49 @@
+use crate::geoip::Location;
+
+/// Calculate the distance between two points using Vincenty's inverse formula.
+/// The distance is returned in meters.
+pub fn geo_get_distance_between(location1: Location, location2: Location) -> f64 {
```

Comment:
considering these two functions are quite location oriented, no, shouldn't we implement these as a methods for 
`geoip::Location`?

something like:
```rust
pub fn get_distance_to(&self, location: Location) -> f64 {
// or if we wanna be inspired by geoutils API:
pub fn distance_to(&self, location: Location) -> f64 {
```
```rust
pub fn is_in_radius(&self, location: Location) -> f64 {
// or
pub fn is_within_radius(&self, location: Location) -> f64 {
```


usage:
```rust
        let distance = zagreb.distance_to(kyiv);
        assert_eq!(distance, 1197101.314);
```

Reply:
Done.